rskj-core/src/test/java/co/rsk/peg/ReleaseTransactionBuilderTest.java

```diff
@@ -904,6 +909,159 @@ void test_VerifyTXFeeIsSpentEquallyForBatchedPegouts_three_pegouts() {
         Assertions.assertEquals(inputsValue.minus(totalPegoutAmount), changeOutput.getValue());
     }
 
+    @ParameterizedTest()
+    @MethodSource("providePegoutMinimumParameters")
+    void buildBatchedPegouts_whenSpendableUXTO_shouldBuildTransactionSuccessfully(Coin feePerKb, Coin minPegoutValue) {
+        // build federation with 9 members
+        Integer[] memberPKs = new Integer[]{ 100, 200, 300, 400, 500, 600, 700, 800, 900 };
+        Federation fed = FederationTestUtils.getFederation(memberPKs);
```

Comment:
This is creating a standard multisig federation. We should test this with an ERP fed, that is the one used at the moment. Larger redeem script since it includes the emergency multisig. You can use FederationFactory class to build it

Reply:
It should have 9 members and 4 in the emergency multisig

Reply:
you can check [P2shErpFederationBuilder](https://github.com/rsksmart/rskj/blob/federation-support-refactor-integration/rskj-core/src/test/java/co/rsk/peg/federation/P2shErpFederationBuilder.java) as a guidance

Reply:
I have updated to use the defaults of [P2shErpFederationBuilder](https://github.com/rsksmart/rskj/blob/federation-support-refactor-integration/rskj-core/src/test/java/co/rsk/peg/federation/P2shErpFederationBuilder.java), let me know if this looks good